### PR TITLE
Movie details

### PR DESCRIPTION
**Pull request by BoAneta**
Now you can click on the movie and website path will change.
I have also moved MovieDetails folder outside the MoviesList.

### CODE REVIEW

**Review by MateuszPiaszczyk — APPROVED** [review]
Overall it looks ok, but for me there are a few minor things to think about :)

File: src/features/MovieDetails/movieDetailsSlice.js [hunk 1]
```diff
@@ -8,13 +8,13 @@ const movieDetailsSlice = createSlice({
...
+        fetchMovieDetails: (_,{payload: movieId}) => ({
             movieInfo: "loading",
             status: null,
-            id
+            movieId
             
         }),
-        fetchMovieDetailsSuccess:(_,{payload:movieInfo}) => ({
+        fetchMovieDetailsSuccess:(_,{payload: movieInfo}) => ({
             status:"success",
```
[the remark this answers is not in the document]
Comment (Dawid-Zieba) [reply src/features/MovieDetails/movieDetailsSlice.js:18]: It's important part, let's add it

File: src/core/App/routes.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export const toMovies = () => "/movies";
 export const toPeople = () => "/people";
-export const toMovie = ({ id } = { id: ":id" }) => `/movies/${id}`;
-export const toPerson = ({ id } = { id: ":id" }) => `/people/${id}`;
+export const toMovie = ({ movieId } = { movieId: ":movieId" }) => `/movies/${movieId}`;
+export const toPerson = ({ personId } = { personId: ":personId" }) => `/people/${personId}`;
```
[the remark this answers is not in the document]
Comment (Dawid-Zieba) [reply src/core/App/routes.js:4]: I think we should stay with Aneta version, id is assigned to the name and will be known

[the remark this answers is not in the document]
Comment (MateuszPiaszczyk) [reply src/core/App/routes.js:4]: ok so be it 👍

**Review by MateuszPiaszczyk — APPROVED** [review]

**Pull request merged**